api/refresh_continuous_aggregate.md

```diff
@@ -62,6 +62,12 @@ not take place when buckets are materialized with no data changes or with
 changes that only occurred in the secondary table used in the JOIN.
 </Highlight>
 
+### Optional arguments
+
+|Name|Type|Description|
+|-|-|-|
+| `force` | BOOLEAN | Force the refresh even if the window range is already refreshed. Default to `FALSE`.|
```

Comment:
Might be good to emphasize that it is an expensive operation, e.g., something like this:
```suggestion
| `force` | BOOLEAN | Force refresh of every bucket in the time range even if the bucket is already refreshed. Can be very expensive if a lot of data is refreshed. Default to `FALSE`.|
```